### PR TITLE
Multi-module doc: disambiguate root vs graphql-root module

### PR DESCRIPTION
**Pull request by martinbonnin**
Add some explanations about the GraphQL root module in the context of multi-module as Gradle also has its own "root" project.

Let me know what you think and I'll forward port to dev-3.x if/once it looks ok.

### CODE REVIEW

File: docs/source/advanced/multi-modules.mdx [hunk 1]
```diff
@@ -9,16 +9,18 @@ Support for multi-modules build is currently experimental and we'd love to have
 
 ## Setup
 
-Configure your parent module to generate Apollo metadata:
+Multi-modules requires that one module and only one contains a schema. This is the schema that all other modules can reuse. In this document, we'll name it "graphql-root". This is **not to be confused with Gradle root project** that contains all your modules.
```
Comment (BoD) [inline docs/source/advanced/multi-modules.mdx:12]:
```suggestion
Multi-modules requires that one and only one module contains a schema. This is the schema that all other modules can reuse. In this document, we'll name it "graphql-root". This is **not to be confused with Gradle root project** that contains all your modules.
```

**Review by BoD — COMMENTED** [review]
I'm not 100% sure "root" is the best term if other modules must depend on it (sometimes in a dependency tree the "root" is the app module, the one who depends on the other ones).

Could we name it the "schema module"?

**Comment by martinbonnin** [comment]
> I'm not 100% sure "root" is the best term if other modules must depend on it (sometimes in a dependency tree the "root" is the app module, the one who depends on the other ones).
> 
> Could we name it the "schema module"?

Agreed. All the things are renamed

File: docs/source/advanced/multi-modules.mdx [hunk 2]
```diff
@@ -27,37 +29,37 @@ dependencies {
...
+// This module can use fragments and types from 'schema'
 dependencies {
-    apolloMetadata(project(":root"))
+    apolloMetadata(project(":schema"))
     generateApolloMetadata.set(true)
 }
 
-// root/build.gradle.kts
-// This module is the root module
+// schemabuild.gradle.kts
```
Comment (BoD) [inline docs/source/advanced/multi-modules.mdx:61]:
```suggestion
// schema/build.gradle.kts
```

**Review by BoD — APPROVED** [review]
👍

**Review by joeldenke — APPROVED** [review]
Looks good, less confusing of root gradle :)

**Pull request merged**